public/static/js/projects.js

```diff
@@ -183,6 +183,31 @@ mciModule.controller('ProjectCtrl', function ($scope, $window, $http, $location,
 
   $scope.refreshTrackedProjects($window.allTrackedProjects);
 
+  $(function  () {
+    $("#workstation-list").sortable({
+      group: 'workstation_list',
+      pullPlaceholder: false,
+      // set $item relative to cursor position
+      onDragStart: function ($item, container, _super) {
```

Comment:
Not sure if there's a dragEnd event, but something should mark the form as dirty if you move commands around